app/graph/types/query_type.rb

```diff
@@ -202,6 +202,18 @@ def dynamic_annotation_field(query:, only_cache: nil)
     end
   end
 
+  field :tipline_request,
+        TiplineRequestType,
+        description:
+          "Information about tipline request, given its id",
+        null: true do
+    argument :id, GraphQL::Types::String, required: true
+  end
```

Comment:
Check permissions.

Reply:
Used same permission for `dynamic_annotation_field` calling.

Reply:
I think that permission only applies to global API keys, so it won't work from the web client for regular users, right?

Reply:
Why we need this API endpoint anyway? We always we want to get the requests under the scope of an item, no?

Reply:
Just talked to Sawy - we can delete this block.

Reply:
Deleted